bundle-workflow/infra/lib/distribution-build-stack.ts

```diff
@@ -0,0 +1,100 @@
+import { CloudFrontAllowedMethods, CloudFrontWebDistribution, OriginAccessIdentity } from '@aws-cdk/aws-cloudfront';
+import {
+  ArnPrincipal, CanonicalUserPrincipal, IRole, PolicyStatement, Role,
+} from '@aws-cdk/aws-iam';
+import { Bucket } from '@aws-cdk/aws-s3';
+import {
+  Arn, CfnOutput, CfnParameter, Construct, Stack, StackProps,
+} from '@aws-cdk/core';
+
+export interface DistributionBuildStackProps extends StackProps {
+  name: string,
+  bucketName?: string,
```

Comment:
These optional parameters are in place so we can deploy in an account that already has used the cloud formation template that I removed